language-server/modules/langserver-core/src/test/resources/completion/query_expression/config/query_expr_ctx_config4.json

```diff
@@ -6,177 +6,21 @@
   "source": "query_expression/source/query_expr_ctx_source4.bal",
```

Comment:
@nadeeshaan @IMS94 Are `expression_context/source/table_constructor_expr_ctx_source4.bal` and this one even valid scenarios? There are syntax errors prior to given cursor position right.

Reply:
@lochana-chathura It is a valid scenario. At LS we focus on providing completion suggestions, etc. Therefore, our tests mostly show intermediate code (which will probably have compilation errors since the code is incomplete). In the mentioned scenario user is writing the `table syntax` and user is filling the `key` (`key<module1://cursor//>`) part and cursor is at `//cursor//`. Therefore, this is a valid scenario when the writing pattern is concerned. 
```
import ballerina/module1;

function testFunction() {
    int[] list1 = [1, 2];
    int value2 = 20;
    
    var test = table key<module1:>

    int value3 = 1;
}
```

Reply:
@IMS94 In both [table-constructor-expr](https://ballerina.io/ballerina-spec/spec.html#table-constructor-expr) and [query-expr](https://ballerina.io/ballerina-spec/spec.html#query-expr), `table` keyword is followed by 
[key-specifier](https://ballerina.io/ballerina-spec/spec.html#key-specifier) := `key` `(` [ `field-name` (`,` `field-name`)* ] `)`.

But, here we have a `type-parameter` after the `key` keyword?

Reply:
@lochana-chathura You are correct. We have to change this test and our completion logic as well. Thanks! I'll look into that.

Reply:
great. thanks!